cmd/changelog/main.go

```diff
@@ -17,11 +17,13 @@ import (
 
 // norevive
 const (
-	BUGFIX      = "Bugfix"
-	ENHANCEMENT = "Enhancement"
-	OTHER       = "Others"
-	BACKPORTING = "Backporting"
-	MISC        = "Misc"
+	BUGFIX          = "Bugfix"
+	DATAENHANCEMENT = "Data Enhancement"
+	ENHANCEMENT     = "Enhancement"
```

Comment:
I think we can remove this.

Reply:
It is needed for the regex so that Enhancement category PRs can be collected and then generated in Data Enhancement.

Reply:
Ahh OK I see...